config/locales/en.helpers.yml

```diff
@@ -69,19 +69,31 @@ en:
     have_fun: "Good luck with your training!"
     closing: "%{appname} Team"
   match_mailer:
-    next_game: "your match (%{team_home} vs. %{team_away}) in the event %{name}"
+    start_time: "When? On %{start_date} at %{start_time} Uhr"
+    place: "Where? %{place}"
+    here: "Here"
+    apply_changes: "%{match_link} you can apply changes."
+    add_results: "%{match_link} you can add results."
     match_notification:
-      info: "begins tomorrow at %{start_time}. You will meet at this place: %{place}"
-      subject: "Your game begins tomorrow"
+      subject: "Your match against %{team_name} in the event %{event_name} starts tomorrow."
+      info_html: "your match against %{team_link} in the event %{event_link} starts tomorrow."
+      info_txt: "your match against %{team_name} in the event %{event_name} starts tomorrow."
+      have_fun: "Have fun!"
     match_canceled:
-      info: "has been canceled."
-      subject: "Your match has been canceled"
+      subject: "Your match against %{team_name} in the event %{event_name} was canceled."
```

Comment:
See comment to (added) line 85 of `de.helpers.yml` regarding full stops (.) at the end of subjects